pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -558,6 +567,23 @@ var AzureDiskVolumeFilter = VolumeFilter{
 	},
 }
 
+// CinderVolumeFilter is a VolumeFilter for filtering Cinder Volumes
```

Comment:
Please add some comments on when you want to remove these cloud provider specific code and rely on CSI plugin.